code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -127,6 +118,14 @@
 		src.modules += new /obj/item/weapon/bonesetter(src)
 		src.modules += new /obj/item/weapon/circular_saw(src)
 		src.modules += new /obj/item/weapon/surgicaldrill(src)
+		src.modules += new /obj/item/weapon/gripper/medical(src) //Allows the Medborg to manipulate organs/iv bags/patches/pills and biomass
+		src.modules += new /obj/item/weapon/matter_decompiler(src) //For cleaning and accumulating biomass to deposit at the cloner
+		src.modules += new /obj/item/weapon/soap(src) //For cleaning the disgusting mess that is Medbay
+		src.modules += new /obj/item/weapon/crowbar(src) //For accessing patients in depowered areas

```

Comment:
Crowbars are shit weapons that deal 5 brute. Circular saws are the weapon of choice.

And yes they do. I'm sorry, I don't have time to wait for someone to oblige me when a patient is dying behind a depowered door.